src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,45 @@
 
 class Animal {
   // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
+
+  checkHealth() {
```

Comment:
this method should belong to class not to its instances. Make it static

Reply:
<img width="799" alt="image" src="https://github.com/user-attachments/assets/1d9c8e3a-7249-446c-af59-3205f7e5a4ba">
still not fixed